common/test/data/manual-testing-complete/problem/45c317cb93d447f293ce982a2eccd77d.xml

```diff
@@ -1,31 +1,3 @@
 <problem attempts="10" weight="null" max_attempts="null" markdown="null">
-  <script type="loncapa/python">
-
-def two_d_grader(expect,ans):
-  import json
-  ans=json.loads(ans)
-  if "ERROR" in ans["protex_answer"]:
-    raise ValueError("Protex did not understand your answer... try folding the protein")
-  return ans["protex_answer"]=="CORRECT"
-
-
-</script>
-  <text>
-    <customresponse cfn="two_d_grader">
-      <designprotein2dinput width="855" height="500" target_shape="W;W;W;W;W;W;W"/>
```

Comment:
@ormsbee This manual test problem is why I thought that the `designprotein2dinput` was the input type for FoldIt